docs/gtm-interchain.md

```diff
@@ -0,0 +1,73 @@
+**Introduction**
+
+IBC and Hub go-to-market is a complex organizational, developmental, and marketing undertaking that involves many entities, software projects, and services. IBC is also intimately bound up to the roadmap, development, and go-to-market of Gaia. Though it is possible to bring IBC to market without Gaia, Gaia should be the primary vehicle and product we are concerned with, since the value of Gaia stems from the utility it brings to the ecosystem of chains connected to and benefiting from its position as a Hub. Thus, this plan will consider go-to-market needs for IBC and Gaia simultaneously. The roadmap will be designed to ensure that IBC is valuable to the Hub and its users, including the many interconnected sovereign blockchains that derive utility from the Hub.
+
+The IBC and Gaia go-to-market initiatives will be about direct engagement with the products that directly validate the utility of the Hub and IBC. This initiative encompasses the utility of various interoperable and cross-chain Cosmos entities such as application-specific blockchains, Bridges, Relayers, Validators, etc. More importantly, we envision that the market will quickly grasp the meaning and relevance of cross-chain interoperability of which Cosmos is a leading contender.
+
+**How we work**
+
+These discussions will take place during weekly Gaia team calls on Fridays. The Hub will determine the value proposition of IBC and how it is integrated into Gaia and how it will serve the larger ecosystem of interconnected blockchains. These discussions will continue to take place with key stakeholders during the bi-weekly Cosmos multi entity call. Importantly, the purpose of the bi-weekly multi entity call will be to consider all technical aspects of IBC launch from a strategic perspective (outlined in &#39;What&#39;s Needed to Successfully Go to Market?&#39;). This will inform the agenda.
+
+Leading up to Stargate, we can consider that Gaia Development was led by Chris Goes (Interchain Berlin) and go-to-market was led by Zaki (Iqlusion). Moving forward, we are aiming to organize Gaia Development led by Shahan Khatchadourian (Interchain Berlin) and go-to-market led by Jelena Djuric (Informal Systems). Both Shahan and Jelena are being onboarded into these roles with the support of the previous leads.
```

Comment:
CG was focused on IBC, and we can claim that Gaia didn't have a maintainer.

Reply:
Can we call me the Cosmos Hub Lead, which includes Gaia plus other activities surrounding Gaia and Cosmos Hub?

Reply:
I don't see a problem with that since it is getting quite tedious to keep referring to development as Hub + IBC, versus just calling it one thing. Cosmos Hub Lead it is